wg-notes/fxl-a11y/index.html

```diff
@@ -884,7 +884,7 @@ <h3>Reading system accessibility requirements</h3>
             <section id="rs-a11y-requirements-metadata">
                 <h4>Display the accessibility metadata</h4>
 
-                <p>A reading system should provide information on accessibility metadata provided by the EPUB file in the OPF. Guidance for key information and proposed wordings are provided by the W3C Publishing Community Group Accessibility taskforce report: <a href="https://www.w3.org/2021/09/UX-Guide-metadata-1.0/principles/">User Experience Guide for Displaying Accessibility Metadata</a>.</p>
+                <p>A reading system should provide information on accessibility metadata provided by the EPUB file in the OPF. Guidance for key information and proposed wordings are provided by the W3C Publishing Community Group Accessibility taskforce report: <a href="https://www.w3.org/community/reports/publishingcg/CG-FINAL-a11y-display-guidelines-20250422/">User Experience Guide for Displaying Accessibility Metadata</a>.</p>
```

Comment:
Again, I wouldn't use a dated version here. Use the latest link url. It's also still referring to the document by the 1.0 name.

```suggestion
                <p>A reading system should provide information on accessibility metadata provided by the EPUB file in the OPF. Guidance for key information and proposed wordings are provided by the W3C Publishing Community Group Accessibility taskforce report: <a href="https://www.w3.org/publishing/a11y/metadata-display-guide/guidelines/">Accessibility Metadata Display Guide for Digital Publications</a>.</p>
```

Reply:
This one still has the dated url. It should be https://www.w3.org/publishing/a11y/metadata-display-guide/guidelines/